app/managers.py

```diff
@@ -0,0 +1,53 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
```

Comment:
In my opinion, better to change the attribute to private
Because someone will do:
manager = ActorManager()
manager.table_name = "no_real_table"
manager.all()
And code stop totally works